task8/main.js

```diff
@@ -0,0 +1,73 @@
+const arr = [1, 2, 3, 4];
+const arr2 = [[1, 2, 3], [4, 5, 6], [7, 8, 9]];
+const arr3 = 
+[
+    [
+        [1, 2, 3], [4, 5, 6], [7, 8, 9]
+    ],
+    [
+        [10, 11, 12], [13, 14, 15], [16, 17, 18]
+    ],
+    [
+        [19, 20, 21], [22, 23, 24], [25, 26, 27]
+    ],
+];
+
+/**
+ * Найти размерность массива
+ * @param { any[] } arr - массив
+ * @returns { number } Размерность массива
+ */
+function getDimension(arr) {
+    if (Array.isArray(arr)) {
+      return 1 + getDimension(arr[0]);
```

Comment:
more preferable is to move some numbers or string to separate constants like `const startIndex = 1;`